### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by pigstake**
@mikehmorrissey @nrswolf 

Review is unfortunately optional I think. Take a look if you can/want.

### CODE REVIEW

File: client/assets/app/stylesheets/_shame.scss [hunk 1]
```diff
@@ -1,41 +1,259 @@
...
+$color-yellow5: rgb(247,232,89);
+$color-yellow6: rgb(249,229,38);
+$color-yellow7: rgb(247,221,23);
+$color-yellow8: rgb(249,215,22);
+
+// Utilities
+.u-clearfix {
+  @include scut-clearfix;
+}
+.u-margin-leader {
```
Comment (mikehmorrissey) [inline client/assets/app/stylesheets/_shame.scss]: @pigstake why the different format for these two?

File: client/assets/app/stylesheets/_shame.scss [hunk 2]
```diff
@@ -1,41 +1,259 @@
...
+  //   background-image: true,
+  //   background-options: false,
+  //   box-sizing: true,
+  //   clearfix: false,
+  //   rem: true,
+  // )
 );
-$votes: (
+
+$ideas: (
```
Comment (mikehmorrissey) [inline client/assets/app/stylesheets/_shame.scss]: @pigstake interesting i've never seen this format before

File: client/assets/app/stylesheets/_shame.scss [hunk 3]
```diff
@@ -1,41 +1,259 @@
...
+  font-size: $fontSize-big3;
+  margin-bottom: -.5rem;
+}
+input[type="text"],
+input[type="number"],
+textarea {
+  @extend .font-N4;
+  @extend .u-margin-trailer;
+  @include transition(background-color $transition-time $transition-ease, border $transition-time $transition-ease);
+  background-color: mix($color-white, $color-yellow1, 90%);
```
Comment (mikehmorrissey) [inline client/assets/app/stylesheets/_shame.scss]: @pigstake do you think mixtures/darkens/lightens should be move to a color variable as well or just inline?

File: client/modules/add-idea/add-idea.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+Template.addIdea.helpers({
+  isBoardOpen: function() {
+    return !!globals.isBoardOpen();
```
Comment (mikehmorrissey) [inline client/modules/add-idea/add-idea.js]: @pigstake feel like you don't need the double !! here. ```globals.isBoardOpen()``` should already be returning a boolean

File: client/modules/add-idea/add-idea.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  }
+});
+
+Template.addIdea.events({
+  'submit .addIdea-form': function(e) {
+    e.preventDefault();
+
+    Ideas.insert({
+      boardId: globals.currentBoard()._id,
+      body: $('.addIdea-body').val(), // man what great selectors
```
Comment (mikehmorrissey) [inline client/modules/add-idea/add-idea.js]: @pigstake lol remove

File: client/modules/add-idea/add-idea.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+  isBoardClosed: function() {
+    return !globals.isBoardOpen();
+  }
+});
+
+Template.addIdea.events({
+  'submit .addIdea-form': function(e) {
+    e.preventDefault();
+
```
Comment (mikehmorrissey) [inline client/modules/add-idea/add-idea.js]: @pigstake would be nice to check to see if there is even a body to send. 
```javascript
Template.addIdea.events({
  'submit .addIdea-form': function(e) {
    e.preventDefault();
    var $ideaBody = $('addIdea-body')
   
    if($ideaBody.val().length) {
      ...insert
      ...clear
    }
  }
})

File: client/assets/app/stylesheets/_shame.scss (hunk 1 above)
Comment (pigstake) [inline client/assets/app/stylesheets/_shame.scss]: @mikehmorrissey different format? you mean the hyphen between margin and leader?

File: client/assets/app/stylesheets/_shame.scss (hunk 2 above)
Comment (pigstake) [inline client/assets/app/stylesheets/_shame.scss]: @mikehmorrissey Sass maps! https://viget.com/extend/sass-maps-are-awesome

I'm not 100% comfortable with them, but Susy makes good use of them.

File: client/modules/idea/idea.html [hunk 7]
```diff
@@ -0,0 +1,14 @@
+<template name="idea">
+    <article class="idea">
+      <section class="idea-content">
+        <p class="idea-body">
+          {{ body }}
```
Comment (mikehmorrissey) [inline client/modules/idea/idea.html]: @pigstake semantic :sparkles:

File: client/assets/app/stylesheets/_shame.scss (hunk 3 above)
Comment (pigstake) [inline client/assets/app/stylesheets/_shame.scss]: @mikehmorrissey Not sure. I went very gung-ho with variable specificity on VenMetro and am recoiling a bit now to find a middle path.

File: client/modules/idea/idea.html (hunk 7 above)
Comment (mikehmorrissey) [inline client/modules/idea/idea.html]: @pigstake i want to start getting better about that myself

File: client/modules/add-idea/add-idea.js (hunk 4 above)
Comment (pigstake) [inline client/modules/add-idea/add-idea.js]: @mikehmorrissey Interesting. I don't think I added that. Not sure why it's showing up. I'll defer to Russell on this one. The states need updating anyway.

File: client/modules/ideas/_ideas.scss [hunk 8]
```diff
@@ -0,0 +1,19 @@
+@include with-layout($ideas) {
```
Comment (mikehmorrissey) [inline client/modules/ideas/_ideas.scss]: @pigstake oh i see now. this is the config you defined above

File: client/modules/add-idea/add-idea.js (hunk 5 above)
Comment (pigstake) [inline client/modules/add-idea/add-idea.js]: @mikehmorrissey hahaha that comment is from @nrswolf and predates my truly great selectors :stuck_out_tongue:

File: client/modules/new-board/new-board.html [hunk 9]
```diff
@@ -1,20 +1,20 @@
...
-    <div>
-      <input class="button" type="submit" value="Create board">
-    </div>
-    <!-- <input type="number" placeholder="Votes per user" id="votes-per-user"> -->
-    <!-- <input type="text" placeholder="Your name" id="username"> -->
-  </form>
+      <footer class="u-textCenter">
+        <input type="submit" value="Create board">
+      </footer>
+      <!-- <input type="number" placeholder="Votes per user" id="votes-per-user"> -->
```
Comment (mikehmorrissey) [inline client/modules/new-board/new-board.html]: @pigstake remove if you're not going to use

File: client/modules/site-header/_site-header.scss [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+  border-bottom: solid 1px $color-gray;
+  border-radius: $border-round;
+  margin-bottom: $whiteSpace * 2;
+  @include breakpoint($break-large) {
+    margin-left: $whiteSpace;
+    margin-right: $whiteSpace;
+  }
+}
+.siteHeader-container {
+  @extend .container;;
```
Comment (mikehmorrissey) [inline client/modules/site-header/_site-header.scss]: @pigstake extra ;

File: client/modules/site-header/_site-header.scss [hunk 11]
```diff
@@ -0,0 +1,64 @@
+.siteHeader {
+  border-bottom: solid 1px $color-gray;
+  border-radius: $border-round;
+  margin-bottom: $whiteSpace * 2;
+  @include breakpoint($break-large) {
```
Comment (mikehmorrissey) [inline client/modules/site-header/_site-header.scss]: @pigstake i really like this style of breakpoint. a lot easy to reason w/

File: client/modules/site-header/_site-header.scss [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+    padding-right: 0;
+  }
+}
+.logo {
+  @extend .font-N7;
+  font-size: $fontSize-big1;
+  position: relative;
+}
+.logo-fill {
+  @include scut-circle(6px);
```
Comment (mikehmorrissey) [inline client/modules/site-header/_site-header.scss]: @pigstake how are you feeling about scut so far. seems like a great addition to our utils. I think we need to decide whether we want to use scut mixins inline or wrap them in our utility namespace. the benefit of the second is that the components will be decoupled from scut and easier to use with different utility libs.

File: client/modules/vote/_vote.scss [hunk 13]
```diff
@@ -0,0 +1,16 @@
+.vote {
+  background-color: rgba($color-black, .5);
+  border-radius: $border-round;
+  display: inline-block;
+  height: 2rem;
+  width: 2rem;
+}
+.vote--mine {
+  background-color: $color-black;
+  &:hover {
```
Comment (mikehmorrissey) [inline client/modules/vote/_vote.scss]: @pigstake :+1:

File: client/modules/site-header/_site-header.scss (hunk 12 above)
Comment (pigstake) [inline client/modules/site-header/_site-header.scss]: @mikehmorrissey Yeah I've been inconsistent with that so far, with some re-namespaced and some like this. I like it so far, though I find myself getting caught trying to use Scut and then rolling back to a lighter solution.

Wrapping the mixins sounds good :+1:

File: client/assets/app/stylesheets/_shame.scss (hunk 1 above)
Comment (mikehmorrissey) [inline client/assets/app/stylesheets/_shame.scss]: @pigstake ah nvm was interpreting incorrectly

File: client/modules/add-idea/add-idea.js (hunk 5 above)
Comment (mikehmorrissey) [inline client/modules/add-idea/add-idea.js]: @pigstake haha i figured

**Comment by mikehmorrissey** [comment]
@pigstake :shipit:

**Pull request merged**